### ISSUE TITLE
Formatting code show in hover popups

### ISSUE THREAD

**Issue by titoBouzout** [sublimelsp/LSP#2423]
I'm not really sure how this could work, but would be great if the code snippets shown on hover could have a bit better formatting. Some typescript messages are very hard to parse, well even with formatting xD, but formatted would make the code snippets digestible. Thanks

**Comment by rchl** [comment]
That's more of a request for lsp-typescript or even more for the underlaying server.

**Comment by rchl** [comment]
Closing in favor of https://github.com/sublimelsp/LSP-typescript/issues/243
